From d99728b2cafb3346679cb8733221c41149fdc760 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 9 Mar 2023 22:52:41 +0100
Subject: [PATCH 1029/2871] t0003: GETTEXT_POISON fix, conclusion

In 3c50032ff528 (attr: ignore overly large gitattributes files,
2022-12-01), we backported a patch onto v2.30.* that was originally
based on a much newer version. The v2.30.* release train still has the
GETTEXT_POISON CI job, though, and hence needs `test_i18n*` in its
tests.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 t/t0003-attributes.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t0003-attributes.sh b/t/t0003-attributes.sh
index 5bd9d9832d..57ba303de8 100755
--- a/t/t0003-attributes.sh
+++ b/t/t0003-attributes.sh
@@ -366,7 +366,7 @@ test_expect_success EXPENSIVE 'large attributes file ignored in tree' '
 	dd if=/dev/zero of=.gitattributes bs=101M count=1 2>/dev/null &&
 	git check-attr --all path >/dev/null 2>err &&
 	echo "warning: ignoring overly large gitattributes file ${SQ}.gitattributes${SQ}" >expect &&
-	test_cmp expect err
+	test_i18ncmp expect err
 '
 
 test_expect_success 'large attributes line ignored in index' '
-- 
2.39.3 (Apple Git-145)

